node/lib/TranslatorInfo.ts

```diff
@@ -0,0 +1,26 @@
+
+/**
+ * Information about a translator, as loaded from the translator's manifest.
+ */
+export class TranslatorInfo {
```

Comment:
Should we capture info like below in TranslatorInfo ?
<arg name="advertisementLocalNameFilter" value="^Polar H7*" />